TESTING.md

```diff
@@ -324,7 +324,7 @@ public function yourCustomFunctionNameInHelper($I)
 
 If the function doesn't fit into any existing helper file:
 - create a new file in the `tests/_support/Helper/Acceptance` directory
-- edit the [acceptance.suite.yml](https://github.com/ConvertKit/convertkit-wordpress/blob/main/tests/acceptance.suite.yml) file, adding
+- edit the [acceptance.suite.yml](https://github.com/Kit (formerly ConvertKit)/convertkit-wordpress/blob/main/tests/acceptance.suite.yml) file, adding
```

Comment:
Another place where simple search and replace didn't seem to workAnother place where simple search and replace didn't seem to work